meson_options.txt

```diff
@@ -3,6 +3,8 @@
 
 option('version-tag', type : 'string',
        description : 'override the git version string')
+option('shared-lib-tag', type : 'string',
```

Comment:
I wonder if setting value to `meson.project_version()` here works?

Reply:
Here no, because this is `meson_options.txt` — we can only do static strings. This fallback needs to be handled in the `meson.build` files somewhere.